cli/internal/runcache/runcache.go

```diff
@@ -193,6 +193,7 @@ func (tc TaskCache) RestoreOutputs(ctx context.Context, terminal *cli.PrefixedUi
 		terminal.Output(fmt.Sprintf("cache hit, suppressing output %s", ui.Dim(tc.hash)))
 	case util.FullTaskOutput:
 		logger.Debug("log file", "path", tc.LogFileName)
+		terminal.Output(fmt.Sprintf("cache hit, replaying output %s", ui.Dim(tc.hash)))
```

Comment:
This is still `cli.PrefixedUi`, so the `package:task` prefix will still be there.